_redirects

```diff
@@ -1,69 +0,0 @@
-/taps/sage-intacct /taps/intacct
```

Comment:
Do we know if anything was relying on these redirects? The `_redirects` file is picked up by Netlify and should still work after we get rid of jekyll.

Reply:
cc @alexmarple, @afolson, @pandemicsyn, @tayloramurphy 

Quick check-in on whether we need these redirects? I imagine yes for safety and to not break existing links? 

@rwfeather - How difficult is it to add these redirects also for Gridsome/Vue? I don't think we should block this PR but  if needed, we can create a follow-on issue to add those back before merging `gridsome-main` to `main`.

Reply:
@aaronsteers yes, we need those redirects still. I don't know what would ever cause us to not need them honestly. But we added those fairly recently so we should keep them.

Reply:
@aaronsteers I could have been a little clearer. All we need to do in this case is keep the `_redirects` file (it is a Netlify config and will keep working after the switch). Comment was made when I noticed that it was marked for deletion in this PR.

@tayloramurphy Thanks for clarifying

I'll revert the deletion.